Update Andrew-Harris.md (#1122)
SSWConsulting/SSW.People.Profiles: Andrew-Harris/Andrew-Harris.md

[[imgBadge]]
| ![SQL Server](../badges/Developer-sql-server.png)

> Andrew is a seasoned **Senior Software Architect** with over 25 years of experience modernisation and integrating Telephony, CRM, ERP, Payroll, Accounting, and Banking systems across sectors such as Banking, Finance, Retail, Hospitality, Sporting, People Management, and eCommerce. 

Author: ```suggestion
Andrew is a seasoned **Senior Software Architect** with over 25 years of experience modernising and integrating Telephony, CRM, ERP, Payroll, Accounting, and Banking systems across sectors such as Banking, Finance, Retail, Hospitality, Sporting, People Management, and eCommerce. 

```

[[imgBadge]]
| ![SQL Server](../badges/Developer-sql-server.png)

> Andrew is a seasoned **Senior Software Architect** with over 25 years of experience modernising and integrating Telephony, CRM, ERP, Payroll, Accounting, and Banking systems across sectors such as Banking, Finance, Retail, Hospitality, Sporting, People Management, and eCommerce. 

Author: ```suggestion
Andrew is a seasoned **Solution Architect** with over 25 years of experience modernising and integrating Telephony, CRM, ERP, Payroll, Accounting, and Banking systems across sectors such as Banking, Finance, Retail, Hospitality, Sporting, People Management, and eCommerce. 

```

[[imgBadge]]
| ![SQL Server](../badges/Developer-sql-server.png)

Andrew is a seasoned **Solution Architect** with over 25 years of experience modernising and integrating Telephony, CRM, ERP, Payroll, Accounting, and Banking systems across sectors such as Banking, Finance, Retail, Hospitality, Sporting, People Management, and eCommerce. 

He is a passionate mentor combining his years of experience and his Scrum Master certification (CSM) to help guide developers through the intricacies of Scrum and the software development lifecycle. Blending technical and business acumen with a collaborative ethos, Andrew fosters environments where ideas flourish, and developers thrive, one line of code at a time.

Andrew favourite tools and technologies include:

> * .NET Core

Reviewer: ```suggestion
* .NET
```

* Azure
* Docker

> * Node.Js

Reviewer: ```suggestion
* Node.js
```
